tests/helpers/storageVolume.ts

```diff
@@ -53,3 +53,24 @@ export const saveVolume = async (page: Page, volume: string) => {
   await page.getByRole("button", { name: "Save changes" }).click();
   await page.waitForSelector(`text=Storage volume ${volume} updated.`);
 };
+
+export const migrateVolume = async (
+  page: Page,
+  volume: string,
+  targetPool: string,
+) => {
+  await visitVolume(page, volume);
+  await page.getByRole("button", { name: "Migrate", exact: true }).click();
+  await page
+    .getByRole("dialog", { name: `Choose storage pool for volume ${volume}` })
+    .locator("tr")
+    .filter({ hasText: targetPool })
+    .getByRole("button")
+    .click();
+  await page
+    .getByLabel("Confirm migration")
+    .getByRole("button", { name: "Migrate", exact: true })
+    .click();
+
+  await page.waitForSelector(`text=successfully migrated to ${targetPool}`);
```

Comment:
The check on line 67 checks whether the pool is displayed on the page, which is kind of a check that the page refreshes (because it has to refresh to display the pool). Does that suffice or should I explicitly add a page check?